easybuild/easyblocks/s/suitesparse.py

```diff
@@ -172,8 +184,25 @@ def configure_step(self):
 
         else:
             # after v6.0.0, no option for metis, its own metis is used anyway
-            # nothing to do here, set the CMAKE_OPTIONS in easyconfigs
-            pass
+            # set CMAKE_OPTIONS if it is not specified in easyconfigs
+            # CMAKE_INSTALL_PREFIX is managed by easybuild
+            cmake_options = '-DCMAKE_INSTALL_PREFIX=%s' % self.installdir
+
+            flexiblas = get_software_root('FlexiBLAS')
+            mkl = get_software_root('imkl')
+            openblas = get_software_root('OpenBLAS')
```

Comment:
It's better to check this via the toolchain, like we do in the `scipy` easyblock for example, see https://github.com/easybuilders/easybuild-easyblocks/blob/d9ecd18a60bc1aee0a0273cd2fcfaf8f5f5279a3/easybuild/easyblocks/s/scipy.py#L123-L129

Reply:
Although the code works, I am curious about where are the definitions of `toolchain.FLEXIBLAS` and others. I cannot find them in the toolchain module.

Reply:
They're constants that get defined automagically by EasyBuild framework.

For `FLEXIBAS`, that's via `TC_CONSTANT_FLEXIBLAS` in `easybuild/toolchains/linalg/flexiblas.py`: https://github.com/easybuilders/easybuild-framework/blob/71955250210fa6a3189afe9f9310c2b332c9b47e/easybuild/toolchains/linalg/flexiblas.py#L41C1-L41C22